docs/mercure.md

```diff
@@ -8,5 +8,6 @@ No JS library or SDK required!
 ![Mercure](mercure-hub.png)
 
 To enable the Mercure hub, update the `Caddyfile` as described [on Mercure's site](https://mercure.rocks/docs/hub/config).
+The path to send data as well as the path of the public URL for clients is `/.well-known/mercure`.  When running FrankenPHP inside Docker, the full send URL would look like `http://php//.well-known/mercure` (with `php` being the name of the container running FrankenPHP).
```

Comment:
Actually, for Docker this is more complicated than that because HTTPS needs to be disabled for internal URL etc. Maybe can we just add a like to what we've done in Symfony Docker?

* https://github.com/dunglas/symfony-docker/blob/main/compose.yaml#L6
* https://github.com/dunglas/symfony-docker/blob/main/frankenphp/Caddyfile#L11

Or even copy this example configuration here in a new `Docker` section?